package.json

```diff
@@ -180,6 +180,7 @@
     "localforage": "^1.7.1",
```

Comment:
- [ ] Enhancement:  Handle cloudflare css caching issues?

If cloudflare cache causes issues with the css chunks (I simulate by blocking in devtools):
- For page transitions, it will just be stuck at the blank state.
- If you block the css for modal, the modal just doesn't appear.

In both cases, the top loading bar just gets stuck at 90% (correct behavior for that component).

For js chunks, `lazyImport` tries to detect that and displays a nag to reload.
It arguable which behavior is better, though.  Maybe a hanging loading bar is better than an annoying nag.